atomic_reactor/plugins/pre_check_base_image.py

```diff
@@ -67,13 +61,13 @@ def __init__(self, workflow, check_platforms=False, inspect_only=False,
 
     def run(self):
         """
-        Pull parent images and retag them uniquely for this build.
+        Check parent images to ensure they only come from allowed registries.
         """
         self.manifest_list_cache.clear()
 
         digest_fetching_exceptions = []
-        for nonce, parent in enumerate(self.workflow.dockerfile_images.keys()):
-            if base_image_is_custom(parent.to_str()):
+        for _, parent in enumerate(self.workflow.dockerfile_images.keys()):
```

Comment:
since we are not using the counter portion of enumerate - I would update this even more and just iterate over the keys.